### ISSUE TITLE
Add a component for custom selects

### ISSUE THREAD

**Issue by titouanmathis** [studiometa/ui#7]
It should help to easily create custom `<select>` form elements with accessibility in mind.

See https://codepen.io/titouanmathis/pen/QWGLRGv.

**Comment by titouanmathis** [comment]
The `Menu` component can already manage dropdowns, maybe extend it to synchronize values with a `<select>`.

**Comment by antoine4livre** [comment]
Extend the Menu component is a great idea because it already include the focus trap that is missing in your codepen @titouanmathis

**Comment by perruche** [comment]
This require more work than extending the Menu component to do it properly and respect the WCAG guidelines for combobox.
see: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/
